CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` and `sandstone/VirtualList.VirtualGridList` to set prop value to default when `undefined` is passed for the prop value
```

Comment:
```suggestion
- `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` to have default prop when `undefined` prop is passed
```

Reply:
We can refer `VirtualList` and VirtualGridList` to `sandstone/VirtualList`. :)

Reply:
Thank you. I fixed to use sandstone/VirtualList